### ISSUE TITLE
here are some more new on how etherum dev deny ayment for bounties and cause billions of usd in damages!!!

### ISSUE THREAD

**Issue by deltatoken** [ethereum/go-ethereum#21819]
https://news.bitcoin.com/5-6-million-stolen-as-etc-team-finally-acknowledge-the-51-attack-on-network/

https://www.coindesk.com/ethereum-learn-dao-attack

https://arxiv.org/pdf/1908.04507.pdf 

trust me if this post also get delted i promise to post the poc tomorrow and the guy who chooses to close these issues with 0 knowladge or care about the etherum users will not be the only one to be balmed.

i also have a lawyer firm i work with and will file a lawsuit against you as obviosly i havvent seen any desire to see if you can prevent the damage that will cayse by this new type of atack - ignorance and abandunt is the reason why so many project some larger then u went bankrupt in past years - eithe rway i do hope i get payment for the poc so i can share it and maybe even help you fix it - but by the looks of it i see it isnt going to happen - to all users that will be affected i strongly suggest not to let it slide but file a lawsiuit in your country - whe it is both a crimianl offence / ngelgent and basically scam as you are not seeing my issues as the etherum team closed it before anyone could contact - then you can file a lasuit in yuo country and not where company is registered - there is great logic to that as the people offended should no t be expected to waste more money to start filing lawsuits abroud - many legal predecents for these things - one mostt know is when bank of america scammed people and chose to fire people who nknew something was wrong or just silnce others - the lawsuits the recived were no limited to the us where bank of america was based!
this is not a project tjat casres about users, about legal promises to pay put bounties and thus im suprised ho they made it so far - though keep in mind some of the largest exchanfges closed in past years did exactly the same errorr - i myself also tried to bring to their attention about a bounty since all these platforms share a world of vulnurabilities from so many aspects - sites are usually limited to xss / sql injection and rfi as well as lfi and many more - the ether and similar unsecure crypto have even more vulnurabilities as they simple silence all security specialist and just play games like yuo can see here - take alook of how many times i tried to warn and the redicoulous response i got from the etherum owners.
so now i just advie all users to do waht i said so that at least when tge atack start  (not by me but i can share - when a poc got on google hacking databse and more - usually hundreds of thousand of script kiddies begin atacking - the atack will result in lost fiunds aspecially if u use browser wallets - but also if u use installed wallet which will result in remote code excution (i cant share it now but sufficed to say it is realted to a window feature all computers have - havent tried the pos on mac but it works on mobile and again i cant fully disclose yet how it works or why since i am still waiting to see if indeed a bounty will be payed for finding this - took me and my team 2 weeks to fins it before contacting the etherum network - hopefully this doesnt affect other netwoeks but the scale of this attack is limited to a specific coontract but to all wallet adress and the basic mechanist in the entire etherum blockchain works - so if any of you have echanges - trust me disable the etherum nodes if you self host a cold wallet - as it will result in the wallet being unusable ir even worse all funds stolen in a few minutes!! 
i am really talking to the users to waen them to contact me as the admin of this project keeps removing my comment - even though i t may help you!!!!! if it is removed again i will of course not continue but when the poc hit please dont contact me to come fix it!!!! i gave you pleanty of chances - telling me to resubmit the pos is crazy - and most of all very unproffesional - instead of contacting the people you work with to contact me - you belive i need to chase you??? doe you really think a security proffesional lack money? i am i tring to help from the kindess of my heart and since i dont want to see etherum go offline in a few month - but if thats the additude i gess it is for the best!!!

**Comment by holiman** [comment]
Instead of filing ticket after ticket on go-ethereum, why don't you just send whatever information you think you have to security@ethereum.org or bounty@ethereum.org? 
 
We keep closing the tickets, as tickets on go-ethereum is meant for discussing bugs with go-ethereum, not discuss potential security vulnerabilities.

**Comment by deltatoken** [comment]
> Instead of filing ticket after ticket on go-ethereum, why don't you just send whatever information you think you have to [security@ethereum.org](mailto:security@ethereum.org) or [bounty@ethereum.org](mailto:bounty@ethereum.org)?
> 
> We keep closing the tickets, as tickets on go-ethereum is meant for discussing bugs with go-ethereum, not discuss potential security vulnerabilities.

i think i explained my self quite alot the fact none of you are reading what i wrote shows all - i did - and even posted the respoence - nevermind im starting to think maybe is better if all users realize what this is - a group of dev who make bliions but refuse to pay a 100k bounty which is related to user security - at leaset if you stopped removed this as an issue then you are not commiting feloney when the sht start in a couple of weeks - i too have no time to post the poc in 2 days so after it is posrted usully all hell breaks loose - also impotrant to note the most hack on the google hacking databse refrence diffrent sites with a "shared vulnurabilty -and yet still in a matter of weeks all sites that have usully get exploited by hundreds of users daily. in this case it will 4 sevices that i belive will completly take the entire blockchain down so any wallet that was offline during that time will be affected the second it comes online - i posted it hewre after 0 reponse fro the official link - so i think even if you are one of the owners you have no problem to contact the other staff and ask them why i wasnt payed bounty? 
anyway i really have no desire for bounty - i didnt insult or curse you and yet you chose to that - see you and hopelly im wrong - but in vm test the results were horrifing. we also performed same tests on only 5 nodes and in a closed network - every single device that at some point connected to that single node got his entire file corrupted - same with mobiles and laptops. after which we connected another device with a slightly altred code that was injected to the browser using a like similat to bitly and the second that user opens wallet the keys are sent to remote server, and a screenshot of seed word the next time it is opened - again i cannot elaborate here as i see no point - obviosly you wont apologize or give us some credit for doing this type of work for - any good luck with upcoming havok - DO NOT CONTACT ME WHEN IT START IN A FEW WEEKS - LEARN HOW TO TALK TO PPL WHO TRY HELP AND NOT HOW TO HIDE SECURITY ISSUES - IF U HIDE THEM DOESNT MEAN THEY ARENT THERE~!!!!
